app/client/src/components/shared/CharacteristicsSelect.tsx

```diff
@@ -44,41 +60,98 @@ export function CharacteristicsSelect({
 
   return (
     <div css={selectContainerStyles}>
-      <label css={selectLabelStyles} htmlFor={inputId}>
-        {label}
-      </label>
+      <span css={selectLabelStyles}>
+        Filter by{' '}
+        <GlossaryTerm term="Characteristics">Characteristics</GlossaryTerm>:
+      </span>
       <Select
+        aria-label="Filter by Characteristics"
         components={{ MenuList }}
```

Comment:
The `inputId` is necessary if the `label` element is retained:
```suggestion
        components={{ MenuList }}
        inputId={inputId}
```